.env.example

```diff
@@ -24,13 +24,11 @@ DEFAULT_SMTP_PORT=
 DEFAULT_SMTP_TLS=
 DEFAULT_SMTP_NO_AUTH=
 
-USER_CONFIG_TYPE=file
+USER_CONFIG_TYPE=DB
```

Comment:
Just one small request - can you leave the old version of the example env file? Most people would probably run fine with file configs rather than setting up a db for cypht.